domain/src/main/java/org/sagebionetworks/research/domain/result/data/AnswerResultData.java

```diff
@@ -51,16 +53,16 @@
     public abstract static class Builder<T> {
         public abstract AnswerResultData<T> build();
 
-        public abstract Builder<T> setAnswer(final T answer);
+        public abstract Builder<T> setAnswer(@Nullable final T answer);
```

Comment:
I'm not sure why answer was strictly non-null, and it causes crashes when you de-select the hand selection step, and iOS doesn't enforce this either, because that is how answers look when you skip.

Reply:
ResearchKit.  Allowing null values is a something that I changed b/c it made things simpler to maintain.